slurm/install/rhel.sh

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
+set -e
+INSALLED_FILE=/etc/azslurm-bins.installed
+if [ -e touch $INSALLED_FILE ]; then
+    exit 0
+fi
+
+SLURM_ROLE=$1
+SLURM_VERSION=$2
+OS_VERSION=$(cat /etc/os-release  | grep VERSION_ID | cut -d= -f2 | cut -d\" -f2)
 yum -y install epel-release
 yum -y install munge
-dnf -y --enablerepo=powertools install -y perl-Switch
-if [ $1 == "scheduler" ]; then
-    yum -y install blobs/*.rpm
+if [ "$OS_VERSION" > "7" ]; then
+    dnf -y --enablerepo=powertools install -y perl-Switch
 else
-    yum  -y install $(ls blobs/*.rpm | grep -v slurmdbd)
-fi
\ No newline at end of file
+    yum -y install python3
+fi
+
+
+if [ ${SLURM_ROLE} == "scheduler" ]; then
+    yum  -y install $(ls slurm-pkgs/*${SLURM_VERSION}.el${OS_VERSION}*.rpm)
```

Comment:
Encountering errors in this line.

I believe in install.py when we call this script, it may not necessarily use the same working directory as the cluster-init, (from which install.py is called). 

The error i am getting is this:

`ls: cannot access 'slurm-pkgs/*23.02.0-1.el8.6*.rpm': No such file or directory
usage: yum install [-c [config file]] [-q] [-v] [--version]
                   [--installroot [path]] [--nodocs] [--noplugins]
                   [--enableplugin [plugin]] [--disableplugin [plugin]]
                   [--releasever RELEASEVER] [--setopt SETOPTS]
                   [--skip-broken] [-h] [--allowerasing] [-b | --nobest] [-C]
                   [-R [minutes]] [-d [debug level]] [--debugsolver]
                   [--showduplicates] [-e ERRORLEVEL] [--obsoletes]
                   [--rpmverbosity [debug level name]] [-y] [--assumeno]
                   [--enablerepo [repo]] [--disablerepo [repo] | --repo
                   [repo]] [--enable | --disable] [-x [package]]
                   [--disableexcludes [repo]] [--repofrompath [repo,path]]
                   [--noautoremove] [--nogpgcheck] [--color COLOR] [--refresh]
                   [-4] [-6] [--destdir DESTDIR] [--downloadonly]
                   [--comment COMMENT] [--bugfix] [--enhancement]
                   [--newpackage] [--security] [--advisory ADVISORY]
                   [--bz BUGZILLA] [--cve CVES]
                   [--sec-severity {Critical,Important,Moderate,Low}]
                   [--forcearch ARCH]
                   PACKAGE [PACKAGE ...]
yum install: error: the following arguments are required: PACKAGE
Traceback (most recent call last):
  File "install.py", line 419, in <module>
    main()
  File "install.py", line 396, in main
    run_installer(settings, os.path.abspath(f"{args.platform}.sh"), args.mode)
  File "install.py", line 89, in run_installer
    subprocess.check_call([path, mode, s.slurmver])
  File "/opt/cycle/jetpack/system/embedded/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/opt/cycle/jetpack/system/bootstrap/azure-slurm-install/rhel.sh', 'scheduler', '23.02.0-1']' returned non-zero exit status 2.`

Reply:
for reference, the rpms are there:

[root@slurm301-1-scheduler azure-slurm-install]# pwd
/opt/cycle/jetpack/system/bootstrap/azure-slurm-install
[root@slurm301-1-scheduler azure-slurm-install]# ls slurm-pkgs | grep 23.02
slurm_23.02.0-1_amd64.deb
slurm-23.02.0-1.el7.x86_64.rpm
slurm-23.02.0-1.el8.x86_64.rpm
slurm-devel_23.02.0-1_amd64.deb
slurm-devel-23.02.0-1.el7.x86_64.rpm
slurm-devel-23.02.0-1.el8.x86_64.rpm

(havent pasted the full output)

Reply:
This issue is because OS_VERSION is coming in at 8.6 instead of 8 - I'll push a fix.